audio/chuck/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                chuck
 version             1.4.0.0
-revision            0
+revision            1
```

Comment:
Build errors usually don't require a `revision` increase. Anyone who has it built already do not need to rebuild it.

Reply:
Updated.